taxi/tests/test_forms.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormTests(TestCase):
+    def test_license_number_is_valid(self):
+        license_numbers = [
+            "111", "aaa", "aaa11111", "aaaaaaaa", "AAA11111",
+        ]
+        result = [False, False, False, False, True]
+
+        for index, license_number in enumerate(license_numbers):
+            form = DriverCreationForm(
```

Comment:
Use @pytest.mark.parametrize

Reply:
I try to use @pytest.mark.parametrize, but I catch an error. Can you explain to me what I'm doing wrong please...

![Screenshot from 2023-11-28 00-03-35](https://github.com/mate-academy/py-taxi-service-search-and-tests/assets/130174413/f53e787c-2c3a-4d4a-b2d2-2f1860f8149f)
![Screenshot from 2023-11-28 00-03-49](https://github.com/mate-academy/py-taxi-service-search-and-tests/assets/130174413/47870263-2826-4b0d-9645-cf79fc170bd6)
?